### ISSUE TITLE
Run another notebook from the current one

### ISSUE THREAD

**Issue by nikjuice** [paiqo/Databricks-VSCode#93]
I have faced with an issue running the another notebook. So in the beginning of the our notebook we have something like
%run ../shared/data_platform_globals
When I run this using the extension I got following:
Exception: File `'..shared/data_platform_globals.py'` not found.

Running this from Databricks works without an issue. 
Please let me know if more details needed.

**Comment by gbrueckl** [comment]
Thanks for reporting, this is a known issue and we are already working on it.
I will update you here once its finished

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Oleh Yermokhin ***@***.***>
Sent: Wednesday, September 14, 2022 11:09:11 AM
To: paiqo/Databricks-VSCode ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [paiqo/Databricks-VSCode] Run another notebook from the current one (Issue #93)


I have faced with an issue running the another notebook. So in the beginning of the our notebook we have something like
%run ../shared/data_platform_globals
When I run this using the extension I got following:
Exception: File '..shared/data_platform_globals.py' not found.

Running this from Databricks works without an issue.
Please let me know if more details needed.

**Comment by nikjuice** [comment]
Thank you for quick update!

**Comment by yrossi55tf1** [comment]
Hello @gbrueckl , any other method instead of %run to do the same ?

**Comment by gbrueckl** [comment]
Not as of now, you could only copy&paste the referenced code directly into your notebook

I am afraid you need to wait till this feature is added which will hopefully be in the next few weeks

**Comment by gbrueckl** [comment]
just published v1.1.3 and `%run` should now work!

a known limitation is that it currently only works if the referenced file uses the same language as the notebook (Python/PySpark)

let me know your feedback!

**Comment by yrossi55tf1** [comment]
Ok thanks, so it has to be in the same local folder ! Thanks for explaining

**Comment by gbrueckl** [comment]
could you also post your full `%run` command please
the error message makes me think you use singe quotes around the referenced file, is that true? 
in case you have blanks or any special characters in the referenced file name, you should use double quotes 
(single quotes also do not work on Databricks)

**Comment by gbrueckl** [comment]
I also just added support for `dbutils.notebook.run` in v1.1.4 when running in Live-mode using a `dbws:/` mount in VSCode workspace!

**Comment by gbrueckl** [comment]
closing this as it was implemented and got no negative feedback anymore

**Comment by nikjuice** [comment]
Sorry for the late feedback, it works. Thanks a lot for so quick adding.

**Comment by gbrueckl** [comment]
Very much depends on how you %run command looks
Can you please share the code and also the file/folder structure

Relative paths shoukd work but i could think of an issue with absolute paths

**Comment by THEIOTIDIOT** [comment]
Code : (calling_notebook)
cmd1 : %run databricks-project/util/generic_notebook
cmd2 : doing something with variables from the nested notebook in cmd1

Folder Structure : 
databricks-project
L util
   L generic_notebook
L calling_notebook

**Comment by gbrueckl** [comment]
Can you try using a relative path starting with `. /` instead?
Also, are you using the workspace browser from the Databricks tab or a `dbws:/` mount from the VSCode Explorer?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by THEIOTIDIOT** [comment]
I downloaded the notebooks in question, from the Databricks tab, and it added a 'Workspace' folder to my VSCode project folder. I am opening the local notebook copy and running that notebook on our databricks cluster. When I use '%run /path/to/notebook' it does a relative search for the notebook I'm attempting to run, but to match the functionality of the Databricks editor, it should be an absolute path from the 'Workspace' folder.

If I was to use a relative path it would make things more difficult. We have a ton of notebooks in various folder structures and if we had to use relative paths for each one of those notebooks, it would be a major headache.
